### PR TITLE
Adds maid clothing to the bartenders wardrobe.

### PR DESCRIPTION
**Pull request by Expletive**
:cl:
tweak: The bartender has a new outfit in their wardrobe to match the Maid Cafe sign.
/:cl:

We have the maid cafe bar sign, so I thought I'd go ahead and do this.

### CODE REVIEW

**Comment by Cheridan** [comment]
wow it literally was a slippery slope

goofball was right

**Comment by DoctoreMad** [comment]
Why don't you slip in a slope and land on my _dick_

**Comment by Bawhoppen** [comment]
I don't think we have a real term, but this is the type of creep where eventually everyone just gets all the different types of items, and diminishes their uniqueness

File: code/game/objects/structures/crates_lockers/closets/job_closets.dm [hunk 1]
```diff
@@ -11,16 +11,18 @@
...
+	new /obj/item/clothing/under/maid(src)
 	new /obj/item/clothing/under/rank/bartender(src)
 	new /obj/item/clothing/under/rank/bartender(src)
 	new /obj/item/clothing/accessory/waistcoat(src)
 	new /obj/item/clothing/accessory/waistcoat(src)
 	new /obj/item/clothing/head/soft/black(src)
 	new /obj/item/clothing/head/soft/black(src)
 	new /obj/item/clothing/shoes/sneakers/black(src)
-	new /obj/item/clothing/shoes/sneakers/black(src)
+	new /obj/item/clothing/shoes/laceup(src)
```
Comment (Bawhoppen) [inline code/game/objects/structures/crates_lockers/closets/job_closets.dm]: stealth change

**Comment by Iamgoofball** [comment]
# S L I P P E R Y S L O P E

File: code/game/objects/structures/crates_lockers/closets/job_closets.dm (hunk 1 above)
Comment (Expletive) [inline code/game/objects/structures/crates_lockers/closets/job_closets.dm]: Is it really? Aren't laceups the designated maid shoes?

**Comment by Iamgoofball** [comment]
# I CALLED IT https://github.com/tgstation/tgstation/pull/25417#issuecomment-288795391

**Comment by Bawhoppen** [comment]
goof you literally call out any likely thing then act like you're the next prophet when it happens

**Comment by Jordie0608** [comment]
On the other hand; there's lots of potential costumes for all the different barsigns, adding them to the bartender's wardrobe sounds folly, isn't this what the costume vendor is for?

**Comment by Iamgoofball** [comment]
>make accurate predictions
>point out when they're correct

wheres the problem here

**Comment by octareenroon91** [comment]
>makes inaccurate predictions
>no follow-up

Confirmation bias if I'm not mistaken.

**Comment by Repukan** [comment]
Slibbery slob

**Comment by ShadowDeath6** [comment]
There are probably more examples of goofball being right in cases like this than him being wrong, though.

I think playing dress-up should be in the clown's domain, unless you want to give the bartender an outfit for every bar sign style.

**Comment by FantasticFwoosh** [comment]
Well its just as well we have a zippo lighter and a disposal bin to get rid of it, if we're not being actively hacked/assaulted for entry into the bar storage so greyshirts can get the sawn off & a meme maid suit to fufill thier IC character loadouts or whatever.

**Comment by lzimann** [comment]
Get theatre backstage access and get the costume there. This is pointless

File: code/game/objects/structures/crates_lockers/closets/job_closets.dm [hunk 2]
```diff
@@ -11,16 +11,18 @@
 	new /obj/item/device/radio/headset/headset_srv(src)
 	new /obj/item/device/radio/headset/headset_srv(src)
 	new /obj/item/clothing/head/that(src)
+	new /obj/item/clothing/head/kitty(src)
```
Comment (lzimann) [inline code/game/objects/structures/crates_lockers/closets/job_closets.dm]: Why are you adding this too

**Comment by PKPenguin321** [comment]
Hahahahahahahaha it happened

File: code/game/objects/structures/crates_lockers/closets/job_closets.dm (hunk 2 above)
Comment (Iamgoofball) [inline code/game/objects/structures/crates_lockers/closets/job_closets.dm]: OH MY GOD

Comment (Iamgoofball) [inline code/game/objects/structures/crates_lockers/closets/job_closets.dm]: # OH MY GOD

**Comment by lzimann** [comment]
This won't be merged, these costumes are already in the vendor, so you either:
1) Add theatre backstage access to bartender, it's not really a big deal if he has access to it(technically it's "his" bar after all)
2) Get the access from HoP in game.

**Comment by KorPhaeron** [comment]
@remierichards

**Comment by Iamgoofball** [comment]
do you really want to restart this

**Pull request closed without merging**